### DISCUSSION TITLE
Minor suggestion: include loot table parity for pumpkins/melons

### DISCUSSION THREAD

**Discussion by lummwastaken** [vectorwing/FarmersDelight#866] [Ideas]
Because Farmer's Delight adds a pumpkin slice, it might be a good QOL feature to give pumpkins similar loot behavior to vanilla melons (dropping full melons w/ silk touch, otherwise dropping a randomized amount of slices)

**Comment by Fabsian10000** [comment]
in my opinion it would actually make more sense to reverse your idea, afterall, in reallife noone slices melons when harvesting them

**Comment by vectorwing** [comment]
I considered adding that, but as a knife drop: if mined with a knife, it splits into slices, which are edible and potentially useful in the earlygame. But I wouldn't modify it to always drop slices, as you described.

**Comment by vectorwing** [comment]
This has been added in a recent commit, and should be out on version 1.2.5 for 1.20+. 👍
Pumpkins will now drop 4 Pumpkin Slices when mined with a knife.
